assets/javascripts/kitten/components/buttons/social-button-icon-words.test.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react'
+import {
+  FacebookButtonIconWords,
+  TwitterButtonIconWords,
+  LinkedinButtonIconWords,
+} from 'kitten/components/buttons/social-button-icon-words'
+import { Button } from 'kitten/components/buttons/button'
+import { FacebookIcon } from 'kitten/components/icons/facebook-icon'
+import { TwitterIcon } from 'kitten/components/icons/twitter-icon'
+import { LinkedinIcon } from 'kitten/components/icons/linkedin-icon'
+
+describe('<FacebookButtonIconWords />', () => {
+  describe('by default', () => {
+    const button = shallow(
+      <FacebookButtonIconWords />
+    )
+
+    it('is a <Button modifier="facebook" />', () => {
+      expect(button.find(Button).exists()).toBe(true)
+      expect(button.props().modifier).toBe('facebook')
+    })
+
+    it('has a <FacebookIcon />', () => {
+      expect(button.find(FacebookIcon).exists()).toBe(true)
+    })
+  })
+
+  describe('children prop', () => {
+    const component = mount(
+      <FacebookButtonIconWords>
+        Lorem ipsum…
+      </FacebookButtonIconWords>
+    )
+
+    it('renders a <p>', () => {
```

Comment:
Le test ici s'appellerait plutôt `renders the right children` (parce que ça ne rend pas un `<p>` mais juste ce qui est passé en enfant dans le composant).